test/e2e/tests/from-import-ui.spec.js

```diff
@@ -233,6 +233,13 @@ describe('Metamask Import UI', function () {
         // should remove the account
         await driver.clickElement({ text: 'Remove', tag: 'button' });
 
+        await driver.wait(async () => {
+          const _currentActiveAccountName = await driver.findElement(
+            '.selected-account__name',
+          );
+          return _currentActiveAccountName !== 'Account 5';
+        }, 10000);
```

Comment:
Nit: A commend would help explain why we're doing this.

```suggestion
        // Wait until selected account switches away from removed account
        await driver.wait(async () => {
          const _currentActiveAccountName = await driver.findElement(
            '.selected-account__name',
          );
          return _currentActiveAccountName !== 'Account 5';
        }, 10000);
```

Reply:
We could also consider using the more-concise `waitForSelector`, and matching the expected next account directly rather than looking for "not account 5" and then "account 1" separately.

```suggestion
        // Wait until selected account switches away from removed account to first account
        await driver.waitForSelector({
            css: '.selected-account__name',
            text: 'Account 5'
          },
          { timeout: 10000 },
        );
```

Then if you apply this suggestion, you'll want to delete next few redundant lines as well.